pycroft/model/user.py

```diff
@@ -416,7 +417,7 @@ def active(self, when=None):
         :rtype: bool
         """
         if when is None:
-            now = object_session(self).query(functions.utcnow()).scalar()
```

Comment:
There is a reason, why `object_session` was used to obtain the correct session. Please don't break this.